cluster-provision/gocli/cmd/run_test.go

```diff
@@ -30,10 +30,10 @@ var _ = Describe("Node Provisioning", func() {
 		It("should execute the correct commands", func() {
 			n := nodesconfig.NewNodeLinuxConfig(1, "k8s-1.30", "", "512M", "", false, true, true, true, true, true)
```

Comment:
It may be worth looking at making the provider name more dynamic rather then statically setting `k8s-1.30` - the k8s-1.30 provider will be removed at some point in the future. 

This could be handled in a follow up PR - doesn't have to be included in this fix.